design-tokens/force-base/border-color.yml

```diff
@@ -150,17 +150,17 @@ props:
     value: '{!WHITE}'
     comment: These borders create the faux checkmark when the checkbox toggle is in the checked state.
   COLOR_STROKE_BRAND:
-    value: '{!SCIENCE_BLUE}'
+    value: '{!PICSA_GREEN-1}'
     cssProperties:
     - stroke
     comment: Our product brand blue.
   COLOR_STROKE_BRAND_HOVER:
-    value: '{!SCIENCE_BLUE}'
+    value: '{!PICSA_GREEN-1}'
     cssProperties:
     - stroke
     comment: Hover stroke color for our product brand blue.
   COLOR_STROKE_BRAND_ACTIVE:
-    value: '{!BISCAY}'
+    value: '{!PICSA_CHARCOAL}'
     cssProperties:
     - stroke
     comment: Active stroke color for our product brand blue.
```

Comment:
@NicolaasDieBaas ? As part of my exploration into how SLDS is put together, I made changes to colors and things with no intent of keeping it that way. In this case, I replaced all instances of Biscay with charcoal. We shouldn't worry too much about these for now. But I will look into proper organisation of color schemes as part of another task.